x-pack/plugins/infra/public/pages/metrics/layouts/pod.ts

```diff
@@ -16,35 +17,47 @@ import {
 export const podLayoutCreator: InfraMetricLayoutCreator = theme => [
   {
     id: 'podOverview',
-    label: 'Pod Overview',
+    label: i18n.translate('xpack.infra.podMetrics.podOverviewLabel', {
```

Comment:
Applying the scheme would make this `xpack.infra.metricDetailPage.podMetricsLayout.layoutLabel` with predictable consequences for the ids below.